docs/source/models/causes/other_causes/index.rst

```diff
@@ -82,29 +70,23 @@ States Data
      - 
    * - C
      - excess mortality rate
-     - :math:`CSMR_\text{other causes}` 
-     - CSMR is equal to the EMR because prevalence is equal to 1
+     - 0
+     - 
    * - C
      - disabilty weight
-     - :math:`YLDs_{c294} - \sum_{cid}^{n} YLDs_{cid}`
-     - Where :math:`\sum_{cid}^{n} YLDs_{cid}` is the sum of YLDs due to all specified modeled components (may be causes, sequelae, and/or impairments). List of such IDs must be provided in concept model document.
-   * - ALL
-     - cause specific mortality rate
-     - :math:`ACMR - \sum_{cid}^{n} CSMR_{cid}`
-     - Where :math:`\sum_{cid}^{n} CSMR_{cid}` is the sum of the CSMRs of modeled causes. List of modeled cause IDs must be provided in concept model document.
+     - :math:`YLDs_{c294} - YLDs_{r192} - (YLDs_{c366} - YLDs_{s182,s183,s184}) + \frac{YLD_{c366} - YLDs_{\text{s182,s183,s184}}}{\text{preg_rate}}`
+     - See extended note below
 
 .. note::
 
-  The CSMR/EMR as well as the "disability weight" in the table above may be modified by modeled risk exposures.
+  Note that we are assuming the YLD rate due to all causes other than maternal disorders (including YLDs due to anemia) do not significantly vary between the pregnant/postpartum not pregnant/postpartum populations. We do rescale the rate of maternal disorders from the general population of women of reproductive age to the rate among the pregnant/postpartum population in this equation. 
 
-.. todo::
-  
-  Confirm with the engineers that it will be possible for risk exposures to modify the "disability weight" as stated above 
+  Note that we include YLDs due to maternal disorders in the disability weight of "background morbidity" here despite the fact that maternal disorders is included as a cause of disability within the nutrition optimization pregnancy simulation. This is because in the simulation, simulants acquire pre-COMO-adjusted maternal disorders YLDs over the course of a single timestep in which accumulation of YLDs due to all other causes besides maternal disorders is paused. Therefore, since maternal disorders YLDs are not accumulated during the remainder of the simulation when anemia YLDs are accumulated, anemia YLDs would not be adjusted for comorbidity with maternal disorders unless they were included in the disability weight for background morbidity, which will be present during the timesteps for which anemia YLDs are accumulated
```

Comment:
I don't understand this!  Let's talk it through.

Reply:
I am also confused about this and would appreciate chatting through it.

Reply:
I'm interested as well! I think I kind of follow this, but I'm unclear why it doesn't matter that we are double-counting maternal disorders YLDs overall. And also, it seems like you would have the reverse problem as well: that the maternal disorders YLDs accumulated while everything is paused are not adjusted for those other things.

Reply:
1. We will double-count maternal disorders in the estimate of the total YLD rate in the simulation. However, we only ever really evaluate YLDs _averted_ between scenarios and we will only avert maternal disorders YLDs that occur during the parturition state and the maternal disorders YLDs in the background morbidity will not change, so we will not double count YLDs averted which is really what we are trying to get right.
2. How we are modeling YLDs due to maternal disorders is by taking the already COMO adjusted YLD rate and converting that to a "disability weight" rather than calculating YLDs from a DW directly. Therefore, the maternal disorders YLDs are already COMO adjusted and don't need to be adjusted additionally.

I'll try to add some of this context to the doc to make this more clear!

Reply:
1. Ah, I thought it was something like that. I assume we don't need to report anything like YLDs averted as a proportion of total YLDs? And do we report DALYs in this sim? It seems like we'd slightly change the balance between YLLs averted and YLDs averted here (i.e. averting a death isn't as good if people accumulate more YLDs per unit time). Probably no big deal even if so; just trying to understand the consequences of this double-count.
2. I see! So, we have the same slight limitation going both directions: our YLDs due to anemia aren't taking into account that we have averted some maternal disorders, and our YLDs due to maternal disorders aren't taking into account that we have averted some anemia. Have I got that right?

Reply:
1. Oh, good point, I hadn't fully though through that impacting the balance between YLLs and YLDs averted that you mention there. I am not sure if the (a) influence of COMO adjusting anemia YLDs for maternal disorders or (b) influence of this issue you just raised will be more influential on our results. For this specific model, simulants only live for 6 weeks past their averted death date (they can only die at birth) after spending ~35 weeks or so before birth while they can experience anemia. So I'm thinking that given there will be a pretty small period to accumulate YLDs following death that I would rather capture the COMO adjustment for anemia appropriately. Let me know if you disagree!
2. Yes, exactly!

I think I merged this prematurely to note these additional nuances! I will follow-up to get them included in the write-up.